### ISSUE TITLE
Exporting single visualization from the backend

### ISSUE THREAD

**Issue by dwilches** [RevealBi/Reveal.Sdk#19]
The new backend export functionality supports exporting a whole dashboard as a PDF, but there is no way to export a single visualization (like what we do on the UI side by calling `maximizeVisualization` and then exporting).

Is this something that can be added?

Thanks.

**Comment by brianlagunas** [comment]
Tracked with internal item AB#26343

**Comment by brianlagunas** [comment]
We are working on this now and are proposing the following API.

We are adding a new collection to the `ExportOptions` object named `Visualizations`. If you add one, or more, `VisualizationExport` items to the collection, only those visualizations will be exported. You can decide which viz to export by using either the **Id** or **Title** of the viz.

by **Id**:
```cs
options.Visualization.Add(new VisualizationExport("VIZ_ID"));
```

by **Title**:
```cs
options.Visualization.Add(new VisualizationExport() { Title = "My Viz Title" });
```

How does this feel to you?

**Comment by dwilches** [comment]
Hello,
This solution sounds good.
Thanks!

**Comment by brianlagunas** [comment]
This has been completed and will be available in our upcoming 1.6.1 release.
